docs/angular/guides/misc-ngrx.md

```diff
@@ -1,484 +1,119 @@
+- Registers `EffectsModule.forRoot([])` in the `imports` array for isolation of side effects.
+- Registers `StoreRouterConnectingModule.forRoot()` in the `imports` array for integration with the [Angular Router](https://angular.io/guide/router).
+- Registers `StoreDevtools.instrument()` in the `imports` array for integration with the [Redux Devtools browser extension](http://extension.remotedev.io/).
 
-We can run the generate command for ngrx with the module and onlyEmptyRoot option to only add the StoreModule.forRoot and EffectsModule.forRoot calls without generating any new files.
+You manage separate slices of state using libraries and feature states.
 
-```bash
-ng g @nrwl/angular:ngrx app --module=apps/<appname>/src/app/app.module.ts  --onlyEmptyRoot
-```
+## Feature Workflow
 
-This can be useful in the cases where we don't have a need for any state at the root (or app) level.
+When building new features using NgRx, you want to manage the state from within a separate library. This allows your
+state to be easily shared across other libraries and applications. The steps below go through the workflow of using NgRx within the context of a library.
 
-### Feature
-
-We can run the generate command for ngrx with the module option to create a new feature level store and corresponding pieces needed:
+The example below generates a library to begin a new feature. For this example, `products` is used as the library name.
 
 ```bash
-ng g @nrwl/angular:ngrx products --module=libs/<libname>/src/mymodule.module.ts
+ng g @nrwl/angular:lib products
 ```
 
-We will see the following files created:
-
-```console
-libs/<libname>/src/+state/products.actions.ts
-libs/<libname>/src/+state/products.effects.ts
-libs/<libname>/src/+state/products.effects.spec.ts
-libs/<libname>/src/+state/products.reducer.ts
-libs/<libname>/src/+state/products.reducer.spec.ts
-```
-
-Also, mymodule.module.ts will have StoreModule.forFeature and EffectsModule.forFeature configured
-
-#### Option) `onlyAddFiles`
-
-We can run the generate command for ngrx with the module and `--onlyAddFiles` option to generate files without adding imports to the module.
-
-```console
-ng g @nrwl/angular:ngrx products --module=apps/<appname>/src/app/mymodule/mymodule.module.ts --onlyAddFiles
-```
-
-This can be useful when we want to start building out our state without wiring it up to our Angular application yet.
-
-<br/>
-  
-----
-
-## Learn by Example
+To manage the feature state:
 
-Consider a command to generate a `Comments` NgRx feature set and register it within an application root ngModule.
+- Use the `ngrx` schematic with the feature name in plural form, such as `products`.
+- Provide a path to the `products` library module.
 
 ```bash
-ng g @nrwl/angular:ngrx Comments --root --module=apps/myapp/src/app/app.module.ts
+ng g @nrwl/angular:ngrx products --module=libs/products/src/lib/products.module.ts --directory +state/products --syntax=creators --defaults
```

Comment:
Below in the API docs for the schematics options it shows "Products" in title case, but here it shows lowercase. Which is correct?

Reply:
I've changed the references to lowercase "products" for clarity. It works either way